readme.md

```diff
@@ -84,6 +84,10 @@ By convention, when the artifact is a library, the files are called
   uses cross-validation.
 - `multi-label` Classification for datasets with a
    multi-label (items) objective field.
+- `recursive-feature-elimination` Script to select the n best features for 
+   modeling a given dataset, using a recursive algorithm
+- `name-clusters`  Script to give names to clusters using important field 
```

Comment:
we also have to add the script to the top-level `metadata.json`... and perhaps here we could list it next to the one that names topic models

Reply:
The one that names topic models is not yet here, am I wrong?